src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -162,39 +166,44 @@ Result AudioStreamAAudio::open() {
     mLibLoader->builder_setErrorCallback(aaudioBuilder, oboe_aaudio_error_callback_proc, this);
 
     // ============= OPEN THE STREAM ================
-    {
-        AAudioStream *stream = nullptr;
-        result = static_cast<Result>(mLibLoader->builder_openStream(aaudioBuilder, &stream));
-        mAAudioStream.store(stream);
-    }
+    mAAudioStream = std::make_shared<AAudioStreamOwner>(aaudioBuilder);
+    std::shared_ptr<AAudioStreamOwner> mine(mAAudioStream);
```

Comment:
I don't think you have to create a second reference. The existence of mAAudioStream should prevent the stream from being deleted. Or am I missing something here?